.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,49 @@
+name: Integration Tests
+
+on: [pull_request, push]
+
+jobs:
+  fprime-integration:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+        fprime-version: [NASA-v1.5.0, NASA-v1.5.1, NASA-v1.5.2, NASA-v1.5.3, v2.0.0, devel]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -e .
+      - name: clone fprime and gps-tutorial
+        run: |
+          git clone -b ${{ matrix.fprime-version }} https://github.com/nasa/fprime.git
+          git clone https://github.com/fprime-community/gps-tutorial.git
+      - name: Revert gps-tutorial version if needed
+        if: |
```

Comment:
I wonder if there's any better way to do this. Maybe some sort of map that maps F' version to gpsapp version?

Reply:
Ok, I think I found a better way